lisette/core.py

```diff
@@ -269,11 +252,9 @@ def random_tool_id():
     return f'toolu_{random_part}'
 
 # %% ../nbs/00_core.ipynb
-def mk_tc(func, idx=1, **kwargs):
-    args = json.dumps(kwargs)
-    if callable(func): func = func.__name__
-    id = random_tool_id()
-    return {'index': idx, 'function': {'arguments': args, 'name': func}, 'id': id, 'type': 'function'}
+def mk_tc(func, args, tcid=None, idx=1):
+    if not tcid: tcid = random_tool_id()
+    return {'index': idx, 'function': {'arguments': args, 'name': func}, 'id': tcid, 'type': 'function'}
```

Comment:
These changes make it easy to create tool messages from the tool call summary message (i.e. when the called function and args are strings). 

The downside is that there's a little more effort involved in creating a tool call message when the function and args are symbols.

For example. Here's the current syntax.

```python
mk_tc(simple_add.__name__, json.dumps(dict(a=5, b=7)))
```

vs the original.

```python
mk_tc(simple_add, a=5, b=7)
```